packages/os/early-boot-config.service

```diff
@@ -2,10 +2,14 @@
 Description=Bottlerocket userdata configuration system
 # Need network online to talk to IMDS.
 After=network-online.target apiserver.service
-Requires=network-online.target apiserver.service
+# We don't want to restart the unit the network goes offline or apiserver restarts
+Wants=apiserver.service network-online.target
 # We only want to run once, at first boot.  This file is created by early-boot-config
 # after a successful run.
 ConditionPathExists=!/var/lib/bottlerocket/early-boot-config.ran
+# Prevent manually starting/stopping the service
```

Comment:
We should say "why" rather than "what":

```suggestion
# Block manual interactions with this service, since it could leave the system in an
# unexpected state.
```